src/objective-c/tests/Podfile

```diff
@@ -92,6 +92,10 @@ post_install do |installer|
   installer.pods_project.targets.each do |target|
     target.build_configurations.each do |config|
       config.build_settings['GCC_TREAT_WARNINGS_AS_ERRORS'] = 'YES'
+      if config.name.include?("Test")
```

Comment:
There was a `config.name == 'Test'` check below in this file. Would that work too (maybe tiny bit safer since it's more restrictive :))?

Reply:
updated.